File: android-templates/testData/golden/testNewEmptyViewsActivity_notInRootPackage/Template test module/src/main/java/com/mycompany/myapp/subpackage/MainActivity.java
package com.mycompany.myapp.subpackage;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.mycompany.myapp.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: android-templates/testData/golden/testNewEmptyViewsActivityKotlin_notInRootPackage/Template test module/src/main/java/com/mycompany/myapp/subpackage/MainActivity.kt
package com.mycompany.myapp.subpackage

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.mycompany.myapp.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}